bootloader/lkenv/lkenv_v2.go

```diff
@@ -114,12 +114,21 @@ type SnapBootSelect_v2_recovery struct {
 	 */
 	Recovery_system_status [SNAP_FILE_NAME_MAX_LEN]byte
 
+	/**
+	 * Matrix mapping boot partition to the optional dtbo partition
+	 * This matrix is optional and if not populated dtbo is not extracted
+	 * If defined, dtbo image will be extracted to the partions based on used
+	 * bootimg partition
+	 * [ <bootimg 1 part label> ] [ <dtbo 1 part label> ]
+	 * [ <bootimg 2 part label> ] [ <dtbo 2 part label> ]
+	 */
+	Bootimg_to_dtboimg_matrix [SNAP_RUN_BOOTIMG_PART_NUM][2][SNAP_FILE_NAME_MAX_LEN]byte
```

Comment:
also shouldn't/couldn't this mapping be kind of static, like a replace this substring of the bootimg part label and you get the dtbo part label?

Reply:
it still seems the case that a setting that give two string prefixes would cover the patterns that we expect? storing the prefixes pair like:

/boot_/dbto_/

would allow to know how to turn boot_a into dtbo_a etc